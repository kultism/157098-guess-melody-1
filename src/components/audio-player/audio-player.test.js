import renderer from 'react-test-renderer';
import React from 'react';
import AudioPlayer from './audio-player';

describe(`<AudioPlayer/>`, () => {
  it(`renders correctly`, () => {
    const tree = renderer
      .create(<AudioPlayer src={`path.ogg`} isPlaying={false} onPlayButtonClick={jest.fn()}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
