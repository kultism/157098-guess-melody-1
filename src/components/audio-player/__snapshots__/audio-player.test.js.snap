// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AudioPlayer/> renders correctly 1`] = `
<div
  className="game__track"
>
  <button
    className="track__button track__button--play"
    onClick={[MockFunction]}
    type="button"
  />
  <div
    className="track__status"
  >
    <audio
      src="path.ogg"
    />
  </div>
</div>
`;
